src/Services/Qrda/Helpers/Date.php

```diff
@@ -46,7 +46,7 @@ public function performance_period_end(Mustache_Context $context)
 
     public function current_time(Mustache_Context $context)
     {
-        return DateHelper::format_datetime_gmdate(date('Y-m-d H:i'));
+        return DateHelper::format_datetime(date('Y-m-d H:i'));
```

Comment:
@kchapple again, date format needs to be in the form xxxxxxxx not xxxx-xx-xx
will not pass IG setup this way.

Reply:
The format_datetime function does that, I could probably just remove the parameter